### PR TITLE
Adding many missing logo_URIs in chaininfo json files.

### PR DESCRIPTION
**Pull request by aweigold**
Utilized existing token logos, online sources, and generation of svgs from pngs.

### CODE REVIEW

File: dig/chain.json [hunk 1]
```diff
@@ -53,6 +53,10 @@
       }
     ]
   },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dig/images/dig.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/dig/images/dig.svg"
```
Comment (JeremyParish69) [inline dig/chain.json:58]: IMO that dig svg is way too different than the PNG and should probably be omitted

File: nyx/chain.json [hunk 2]
```diff
@@ -45,6 +45,10 @@
       }
     ]
   },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/nyx/images/nyx.svg"
```
Comment (JeremyParish69) [inline nyx/chain.json:50]: same with this svg, looks like a bad recreation attempt and should probably be omitted

File: mises/chain.json [hunk 3]
```diff
@@ -47,6 +47,10 @@
       }
     ]
   },
+  "logo_URIs": {
+    "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mises/images/mises.png",
+    "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/mises/images/mises.svg"
```
Comment (JeremyParish69) [inline mises/chain.json:52]: These generated svgs are all black; not really accurate...

**Comment by aweigold** [comment]
@JeremyParish69 agreed on that, they are removed... we were working on a project where I needed them, but I'll omit them in this PR.

**Pull request merged**